lib/cosmos/tools/tlm_viewer/widgets/widget.rb

```diff
@@ -200,7 +199,7 @@ def process_settings
             sheet << setting_values[0]
           end
         rescue => err
-          errors << "Error processing settings due to: #{err}"
+          puts "Error Processing Settings!: #{err}"
```

Comment:
We weren't doing anything with the errors return so I just add puts like all the other widgets

Reply:
I don't really like using puts in GUI tools....

Reply:
Honestly there's not anything that can throw an exception here anyway. It's all just string processing. I'll remove the rescue.